src/apps/schema/src/app/views/SchemaCreate/SchemaCreate.js

```diff
@@ -384,12 +393,28 @@ export default connect((state) => {
               navigation and default routing for this model's items.
             </p>
 
-            <FieldTypeDropDown
-              name="parent"
-              label="Select this model's parent"
-              onChange={(value) => setParent(value)}
-              options={props.parents}
-            />
+            <FormControl fullWidth>
+              <FormLabel sx={{ mb: "0 !important" }}>
+                Select this model's parent
+              </FormLabel>
+
+              <VirtualizedAutocomplete
+                name="parent"
+                displayEmpty
+                value={parent}
+                onChange={(_, option) => {
```

Comment:
What is the first parameter provided by the `VirtualizedAutocomplete`?

Reply:
The event